railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_5_2.rb.tt

```diff
@@ -25,3 +25,6 @@
 # Store boolean values are in sqlite3 databases as 1 and 0 instead of 't' and
 # 'f' after migrating old data.
 # Rails.application.config.active_record.sqlite3.represent_boolean_as_integer = true
+
+# Highlight code that triggered database queries in logs.
+Rails.application.config.active_record.verbose_query_logs = Rails.env.development?
```

Comment:
This should not be uncommented.

As we generate it into `development.rb`, I'm actually not sure it should even be here.

Reply:
@matthewd Should I do a follow-up PR to fix this or can someone with a commit bit fix it?

Reply:
If you have the time, a PR would be great; otherwise I'll throw it on my todo list :heart:

Reply:
Oh, looks like @eugeneius already took care of it in #31519